app/src/features/sessionBook/components/SessionViewerBottomSheet/SessionViewerBottomSheet.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from "react";
+import { BottomSheet, BottomSheetPlacement, useBottomSheetContext } from "componentsV2/BottomSheet";
+import { useMediaQuery } from "react-responsive";
+import { useLocation } from "react-router-dom";
+import { useSessionBottomSheetTabItems } from "./hooks/useSessionBottomSheetTabItems";
+
+const BOTTOM_SHEET_TAB_KEYS = {
+  INFO: "info",
+  CONSOLE: "console",
+  NETWORK: "network",
+  ENVIRONMENT: "environment",
+};
+
+interface SessionViewerBottomSheetProps {
+  playerTimeOffset: number;
+  disableDocking?: boolean;
+}
+
+const SessionViewerBottomSheet: React.FC<SessionViewerBottomSheetProps> = ({
+  playerTimeOffset,
+  disableDocking = false,
+}) => {
+  const location = useLocation();
+  const { toggleSheetPlacement } = useBottomSheetContext();
+  const bottomSheetBottomBreakpoint = useMediaQuery({ query: "(max-width: 1092px)" });
+
+  const bottomSheetTabItems = useSessionBottomSheetTabItems({ playerTimeOffset });
+
+  useEffect(() => {
+    if (bottomSheetBottomBreakpoint && location.pathname.includes("saved")) {
```

Comment:
`location.pathname.includes("saved")` this seems like it could break later when the routes might/will change.

Can we change this check to not rely on the location; and if not; can we at least use the route constant here?